package.json

```diff
@@ -53,31 +53,31 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^18.0.1",
-    "@angular/common": "^18.0.1",
-    "@angular/compiler": "^18.0.1",
-    "@angular/core": "^18.0.1",
-    "@angular/forms": "^18.0.1",
-    "@angular/platform-browser": "^18.0.1",
-    "@angular/platform-browser-dynamic": "^18.0.1",
-    "@angular/router": "^18.0.1",
+    "@angular/animations": "^19.0.1",
+    "@angular/common": "^19.0.1",
+    "@angular/compiler": "^19.0.1",
+    "@angular/core": "^19.0.1",
+    "@angular/forms": "^19.0.1",
+    "@angular/platform-browser": "^19.0.1",
+    "@angular/platform-browser-dynamic": "^19.0.1",
+    "@angular/router": "^19.0.1",
     "@schematics/angular": "^18.0.2",
     "rxjs": "^7.4.0",
     "shelljs": "^0.8.5",
     "tslib": "^2.1.0",
-    "zone.js": "~0.14.2"
+    "zone.js": "~0.15.0"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^18.0.2",
-    "@angular-devkit/core": "^18.0.2",
-    "@angular-devkit/schematics": "^18.0.2",
-    "@angular-eslint/builder": "^18.0.0",
-    "@angular-eslint/eslint-plugin": "^18.0.0",
-    "@angular-eslint/eslint-plugin-template": "^18.0.0",
-    "@angular-eslint/schematics": "^18.0.0",
-    "@angular-eslint/template-parser": "^18.0.0",
-    "@angular/cli": "^18.0.2",
-    "@angular/compiler-cli": "^18.0.1",
+    "@angular-devkit/build-angular": "^19.0.2",
+    "@angular-devkit/core": "^19.0.2",
+    "@angular-devkit/schematics": "^19.0.2",
+    "@angular-eslint/builder": "^19.0.0",
+    "@angular-eslint/eslint-plugin": "^19.0.0",
+    "@angular-eslint/eslint-plugin-template": "^19.0.0",
+    "@angular-eslint/schematics": "^19.0.0",
+    "@angular-eslint/template-parser": "^19.0.0",
+    "@angular/cli": "^19.0.2",
+    "@angular/compiler-cli": "^19.0.1",
     "@evilmartians/lefthook": "^1.0.3",
     "@types/jasmine": "^4.0.0",
     "@types/node": "^16.11.7",
```

Comment:
To resolve the build error, the `@types/node` version also needs to be upgraded.

Reply:
Done